Week3/homework/7-step3.js

```diff
@@ -23,3 +23,6 @@ console.log(y);
 
 // Add your explanation as a comment here
 
+// in the first example because it is a primitive data type, it passes by value,
+// but in the second example it is an object and, objects pass by reference.
```

Comment:
👍 
A quick point which I wanted to otherwise a correct answer though - **JavaScript is always call by value. However, in case of objects, the value which is passed is the reference itself.** It doesn't change anything in the way you have learnt things as they behave according to your expectations but just wanted to clarify it.